docs/en/Getting-Started.md

```diff
@@ -1,81 +1,365 @@
-Alluxio also provides following advanced features.
-
-* [Tiered storage](Tiered-Storage-on-Alluxio.html) provides additional resources for Alluxio to
-manage (such as SSD or HHD), allowing for data sets that cannot fit into memory to still take
-advantage of the Alluxio architecture.
-* [Unified and Transparent Namespace](Unified-and-Transparent-Namespace.html) provides the ability
-for users to manage data from existing storage systems and easily handle deployments where not all
-systems are Alluxio-aware.
-* [Lineage](Lineage-API.html) provides an alternative to costly disk replication for fault tolerance
-and data durability, greatly improving write performance.
+The simplest way to quickly try out Alluxio is to install it locally on a single machine. In this
+quick start guide, we will install Alluxio on your local machine, mount example data, and and
+perform basic tasks with the data in Alluxio. During this guide, you will:
+
+* Download and configure Alluxio
+* Start Alluxio locally
+* Perform basic tasks via Alluxio Shell
+* **[AWS Bonus]** Mount a public Amazon S3 bucket in Alluxio
+* Shutdown Alluxio
+
+**[AWS Bonus]** If you have an
+[AWS account with an access key id and secret access key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html),
+you will be be able to perform additional tasks in this guide. Sections of the guide which require
+your AWS account information will be labeled with **[AWS Bonus]**.
+
+## Prerequisites
+
+For the following quick start guide, you will need:
+
+* Mac OS X or Linux
+* Java 7 or newer
+* AWS account and keys
+
+### Setup SSH (Mac OSX)
+
+If you are using Mac OSX, you may have to enable the ability to ssh into localhost. To enable remote
+login, Open **System Preferences**, then open **Sharing**. Make sure **Remote Login** is enabled.
+
+## Downloading Alluxio
+
+First, download the the Alluxio release, and unpack it.
+
+```bash
+$ curl -O http://alluxio.org/downloads/files/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
+$ tar -xzf alluxio-{{site.ALLUXIO_RELEASED_VERSION}}-bin.tar.gz
+$ cd alluxio-{{site.ALLUXIO_RELEASED_VERSION}}
+```
+
+This will create a directory **alluxio-{{site.ALLUXIO_RELEASED_VERSION}}** with all of the Alluxio
+source files and Java binaries.
+
+## Configuring Alluxio
+
+Before we start Alluxio, we have to configure it. We will be using most of the default settings, but
+we will have to add some parameters for accessing Amazon S3 later.
+
+Create the **conf/alluxio-env.sh** configuration file from the template. You can create the config
+file with the following command:
+
+```bash
+$ ./bin/alluxio bootstrap-conf localhost
+```
+
+### [AWS Bonus] Configuration for AWS
+
+If you have an Amazon AWS account with your access key id and secret key, you can modify the Alluxio
+configuration for performing other tasks later in this guide. Open the config file
```

Comment:
reword this sentence so that it's clear that "later in this guide" applies to when the other tasks are performed, not when the configuration should be modified